blog.md

```diff
@@ -0,0 +1,100 @@
+
+Creating and maintaining small, focused, easy-to-review PRs is a comfortable and desired approach
+for both the authors and, especially, the reviewers.
+We created a plugin that eases the management of 2+ branches in the repository
+while also promoting creation of small PRs &mdash;
+**a git repository organizer, and a rebase/merge/push/pull automation tool, named Git Machete.**
+
+It allows for looking at the state of the repository from a previously inexperienceable perspective.
+It puts the primary focus on branches (rather than commits), their relationship to each other,
+and their relationship to the remote repository.
+Moreover, it provides both diagnostic and corrective functionalities
+for the shape of the repository and the mentioned relationships.
+It finds an exceptional application in maintaining linear Git history.
+
+
+### Git Machete Status Graph
+
+![Git Machete Tab with demo repository](gmt_dark.png "Git Machete Tab")
+
+The picture above presents the demo repository _filter-service_ with a few branches.
+It contains two **root branches**: _develop_ and _master_.
+The first of them has two **child branches**: _allow-ownership-link_
+(with another child branch - _build-chain_), and _call-ws_.
+
+Each line contains a name of a branch, an optional custom annotation, and relation to the remote branch.
+
+Without going into details, the **green edge** indicates the desired state -
+the tip of the child branch is a direct descendant of the parent branch tip
+(i.e. all commits reachable from the parent are also reachable from the child).<br/>
+On the other hand, the **red edge** tells us the opposite -
+the tip of the child branch is **not** a direct descendant of the parent branch tip.
+
+In most cases, an edge becomes red after committing some changes to the parent branch (review fixes)
+or pulling new commits to the parent from a remote.
+
+
+### Machete File
+
+![Machete file](machete-file_dark.png "Machete file")
+
+The branches presented within the graph are defined in a very simple configuration file (`.git/machete`),
+which can be easily opened from the plugin's toolbar.
+It supports both syntax highlighting and completion of local branch names.
+The graph is automatically refreshed when `.git/machete` is modified.
+
+Bear in mind that there is no need to edit the machete file manually to set up your branch layout tree.
+Feel free to use `Discover Branch Layout` functionality that will find it out for you automatically.
+It can be performed from `Help -> Find Action...` (default shortcut: `Ctrl + Shift + A`).
+
+
+### Lights, Camera... Actions
+
+The plugin provides a bunch of actions that can help you to manage your repository.
+
+Among the others, there is `Rebase onto parent`
+(that performs interactive rebase of a given branch onto its parent), `Push` and `Pull`.
+Let's have a look at the following example...
+
+![Git Machete - Actions](actions-before_dark.png "Git Machete - Actions - Before")
```

Comment:
Pls replace with a new screenshot (so that action names are more concise)... possibly doesn't matter for other screenshots in this post, though